db/system_keyspace.cc

```diff
@@ -2124,10 +2125,12 @@ static bool maybe_write_in_user_memory(schema_ptr s) {
 future<> system_keyspace::make(
         locator::effective_replication_map_factory& erm_factory,
         replica::database& db) {
+    auto& cfg = _db.get_config();
     for (auto&& table : system_keyspace::all_tables(db.get_config())) {
         co_await db.create_local_system_table(table, maybe_write_in_user_memory(table), erm_factory);
         co_await db.find_column_family(table).init_storage();
     }
+    s_repair_histroy_ttl_seconds = cfg.repair_histroy_ttl_seconds();
```

Comment:
This means the ttl time is set only once during boot up. It is not going to change until next reboot.